generate

```diff
@@ -96,50 +96,15 @@ def gen_menu(structure, override, prefix):
             continue
 
         if len(item['menu']) == 1:
-            if "generator" in item and item['generator'] == "manpages":
```

Comment:
Why is this not needed anymore? Isn't this what generates the sub-menu?

(I don't have the manpages locally to test.)

Reply:
The vanilla menu doesn't support 2nd level menus. I talked to vanilla dev, and he mentioned it is considered bad UX. Hence, I suggest linking to the normal page. Like https://linuxcontainers.org/lxc/manpages//index.html already exists. This is what current PR is doing, and I removed this generation code for 2nd level menus.